e2e/utils/DebugDriver.ts

```diff
@@ -22,6 +22,32 @@ export const setupFork = async (testInfo: TestInfo) => {
   process.env[`TENDERLY_FORK_ID_TEST_${testInfo.testId}`] = fork.id;
 };
 
+export const setupLocalStorage = async (page: Page, testInfo: TestInfo) => {
+  const forkId = process.env[`TENDERLY_FORK_ID_TEST_${testInfo.testId}`];
+  if (!forkId)
+    throw new Error('Fork should be created before setting theRC URL');
+  const rpcUrl = forkRpcUrl(forkId);
+  // We need to be on a page to set localstorage so we create an empty page
+  await page.route(testInfo.testId, (route) => {
+    return route.fulfill({ status: 200, contentType: 'text/plain', body: '' });
+  });
+  await page.goto(testInfo.testId);
+
+  return page.evaluate((rpc) => {
+    // each value is stringified to match lsservice
+    localStorage.setItem('carbon-v1.1-tenderlyRpc', `"${rpc}"`);
+    localStorage.setItem('carbon-v1.1-isUncheckedSigner', 'true');
```

Comment:
Why not set imposterAccount here too and remove the need for debug.setImposter? Would probably save some more time and avoid visiting the debug page altogether for some tests

Reply:
We still need to get funds. The tenderly RPC was the most critical because it requires a reload. We can try to get rid of the need for debug page in another PR.

Reply:
Oh, you're right. Agree with doing it in another PR